sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -31,6 +32,11 @@ def streaming_test():
 def stream_compressed_header_error():
     yield b'test'
 
+def stream_compress():
+    file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", "./files/test.tar.gz"))
```

Comment:
Okay I see the error in CI now: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1318031&view=logs&j=c7500c44-708b-56ec-f52b-02af33ea1457&t=df38eb02-057c-5be2-a959-7d0c1b0db6de&l=3054

It looks like it's not finding the file in CI. I'm not totally sure why the path created locally is different than what we see in CI, but we can try adding a try/except here removing one of the ".." so that it works locally and in CI.